### PR TITLE
BaristaListInteractions enhancements

### PR DESCRIPTION
**Pull request by DeanPike**
Added functionality to find an item in a recycler or list view to BaristaListInteractions.

### CODE REVIEW

**Comment by DeanPike** [comment]
Hi.

Is anyone available to review my pull request, as I need to use it in the project I am currently working on.

Thanks

Dean

File: library/src/main/java/com/schibsted/spain/barista/interaction/BaristaListInteractions.kt [hunk 1]
```diff
@@ -12,25 +12,29 @@ import android.support.test.espresso.action.ViewActions.scrollTo
 import android.support.test.espresso.assertion.ViewAssertions.matches
 import android.support.test.espresso.contrib.RecyclerViewActions.actionOnItemAtPosition
 import android.support.test.espresso.contrib.RecyclerViewActions.scrollToPosition
-import android.support.test.espresso.matcher.ViewMatchers.isAssignableFrom
-import android.support.test.espresso.matcher.ViewMatchers.isDisplayed
-import android.support.test.espresso.matcher.ViewMatchers.withId
+import android.support.test.espresso.matcher.ViewMatchers.*
```
Comment (SmasSive) [inline library/src/main/java/com/schibsted/spain/barista/interaction/BaristaListInteractions.kt]: Please, do not use wildcards

File: library/src/main/java/com/schibsted/spain/barista/interaction/BaristaListInteractions.kt [hunk 2]
```diff
@@ -12,25 +12,29 @@ import android.support.test.espresso.action.ViewActions.scrollTo
...
 import org.hamcrest.Matcher
-import org.hamcrest.Matchers.allOf
-import org.hamcrest.Matchers.anyOf
-import org.hamcrest.Matchers.anything
+import org.hamcrest.Matchers.*
+import org.hamcrest.TypeSafeMatcher
 
 object BaristaListInteractions {
 
+    const val NO_VIEW_ID: Int = -1
```
Comment (SmasSive) [inline library/src/main/java/com/schibsted/spain/barista/interaction/BaristaListInteractions.kt]: I I'm not wrong, could be private right?

File: library/src/main/java/com/schibsted/spain/barista/interaction/BaristaListInteractions.kt [hunk 3]
```diff
@@ -58,6 +62,40 @@ object BaristaListInteractions {
         )
     }
 
+    @JvmStatic
+    fun assertListItemCount(@IdRes listId: Int, expectedItemCount: Int) {
```
Comment (SmasSive) [inline library/src/main/java/com/schibsted/spain/barista/interaction/BaristaListInteractions.kt]: Those assertions should be moved to another class named something like `BaristaListAssertions`. What do you think @Sloy ?

**Comment by DeanPike** [comment]
I have created a new pull request with the requested changes.

**Pull request closed without merging**